### DISCUSSION TITLE
nextjs中包含AntdRegistry ssg生成项目会导致每个html文件都有重复样式

### DISCUSSION THREAD

**Discussion by bosens-China** [ant-design/ant-design#51279] [Q&A]
``` tsx
    <AntdRegistry>
      <ConfigProvider
        locale={zhCN}
        theme={{
          cssVar: true,
          hashed: false,
          token: {
            colorPrimary: colorPrimary,
            borderRadius: 12,
          },
        }}
      >
        <App component={false}>{children}</App>
      </ConfigProvider>
    </AntdRegistry>
```
我有这样一段代码，现在现状是
- 文件1
![image](https://github.com/user-attachments/assets/c70bffbc-cdaa-40da-938a-a4d5bc91113c)

- 文件2
![image](https://github.com/user-attachments/assets/a9e3952e-28b2-4028-afe8-e16b1f039731)

有什么办法可以减少这部分的体积吗？

**Comment by zombieJ** [comment]
它是用到的部分就渲染上去，如果你每个页面样式都是一样的可以考虑直接抽成一个 css 文件。

https://ant-design.antgroup.com/docs/react/server-side-rendering-cn

**Comment by bosens-China** [comment]
@zombieJ 我可以理解这个注入，不过感觉很有强迫症，就是看每个html都有这样一段，你有什么好的建议吗
